pulseapi/settings.py

```diff
@@ -31,10 +31,6 @@
     CORS_ORIGIN_REGEX_WHITELIST=(list, []),
     CORS_ORIGIN_WHITELIST=(list, []),
     CSRF_TRUSTED_ORIGINS=(list, []),
-    AWS_SQS_REGION=(str, None),
```

Comment:
change history of this file: https://github.com/mozilla/network-pulse-api/commits/master/pulseapi/settings.py